lib/node/runner/checker.go

```diff
@@ -594,6 +597,7 @@ func FinishedBallotStore(c common.Checker, args ...interface{}) (err error) {
 		}
 
 		checker.Log.Debug("ballot was stored", "block", *theBlock)
+		checker.NodeRunner.SavingBlockOperations().Save(*theBlock)
```

Comment:
I wonder when the `BlockOperation` is used.
Because in this pr, `BlockOperation`s are not stored immediately after the consensus is over.

Reply:
@spikeekips Exactly :)
Therefore, if some API(It's just my guess) wants to get `BlockOperation` after the consensus immediately, it might be not stored. Is it okay?

Reply:
@Charleslee522 Yes, client will not get `BlockOperation` immediately after consensus, but `SavingBlockOperation` will catch up :)

Reply:
>  client will not get BlockOperation immediately after consensus

@spikeekips Thank you for your kind answer.
Is `client will not get` mean that client can get `BlockOperation` immediately, but don't?
or
client cannot get `BlockOperation` immediately?

Reply:
@Charleslee522 The scenario is,

1. `finishBallot` finishes, but `BlockOperation` will be saved at this time
-> API will not return `BlockOpertion`

2. `finishBallot` calls `SavingBlockOperations().Save()`
-> API will not return `BlockOpertion`

3. after `finishBallot`, `SavingBlockOperations` will start to save `BlockOperation`
-> API will return `BlockOpertion`